pkg/gui/keybindings.go

```diff
@@ -382,6 +382,16 @@ func (gui *Gui) GetKeybindings() []*Binding {
 			Key:      'q',
 			Modifier: gocui.ModNone,
 			Handler:  gui.handleMenuClose,
+		}, {
+			ViewName: "",
+			Key:      'J',
```

Comment:
these keybindings don't seem to work for me. Are they currently working for you?

Reply:
Sorry, I breaked this working feature.